apps/fluent-tester/src/TestComponents/ColorTokens/ColorTokenTest.tsx

```diff
@@ -29,52 +32,72 @@ const styles = StyleSheet.create({
     alignItems: 'center',
     marginVertical: 5,
   },
-  pickerContainer: {
-    flexDirection: 'row',
-    justifyContent: 'space-evenly',
-  },
   stackStyle: {
     borderWidth: 2,
     padding: 12,
     margin: 8,
   },
+  statusView: {
+    flexDirection: 'row',
+    alignItems: 'center',
+    justifyContent: 'space-between',
+    margin: 8,
+  },
+  // Needed to ensure that with the button is toggled on, the top of the newly shown
+  // content is visible so the user knows to scroll down
+  toggleButtonAreaPadding: {
+    paddingBottom: 10,
+  },
 });
 
-const getSwatchColorStyle = (name: string, color: ColorValue): ViewStyle => {
-  styles[name] = styles[name] || { backgroundColor: color };
-  return styles[name];
+const getSwatchColorStyle = (colorName: string, colorValue: ColorValue): ViewStyle => {
+  styles[colorName] = styles[colorName] || { backgroundColor: colorValue };
+  return styles[colorName];
 };
 
-type ColorTokenProps = { color: ColorValue; name: string };
+type ColorTokenProps = { colorValue: ColorValue; colorName: string };
 const ColorToken: React.FunctionComponent<ColorTokenProps> = (p: ColorTokenProps) => {
+  if (p.colorValue === undefined) {
+    console.warn('Color token ' + p.colorName + ' is undefined');
+  }
+
   const themedStyles = getThemedStyles(useTheme());
   return (
     <View style={styles.swatchItem}>
       <View
-        style={[getSwatchColorStyle(p.name, p.color), themedStyles.swatch]}
+        style={[getSwatchColorStyle(p.colorName, p.colorValue), themedStyles.swatch]}
         /* For Android E2E testing purposes, testProps must be passed in after accessibilityLabel. */
         {...testProps(COLORTOKENS_TEST_COMPONENT)}
       />
-      <Text>{p.name}</Text>
+      <Text>{p.colorName + ' (' + p.colorValue?.toString() + ')'}</Text>
     </View>
   );
 };
 
-const AliasTokensSwatchList: React.FunctionComponent = () => {
+const getSwatch = (item) => {
+  const { colorValue, colorName } = item;
+  return <ColorToken key={colorName} colorValue={colorValue} colorName={colorName} />;
+};
+
+/// Alias color tokens
```

Comment:
I think the `///` is something used by Xcode but not in JS code much?

Reply:
We use this in a couple other places - ex. Color.types.ts. Should I change it to something else?

Reply:
Oh. If it's an existing pattern then nvm.